src/seml/manage.py

```diff
@@ -290,6 +296,24 @@ def delete_experiments(
     """
     from seml.console import prompt
 
+    # Before deleting, we should first cancel the experiments that are still running.
+    if cancel:
+        cancel_states = set(States.PENDING + States.RUNNING)
+        if filter_states is not None and len(filter_states) > 0:
+            cancel_states = cancel_states.intersection(filter_states)
+
+        if len(cancel_states) > 0:
+            cancel_experiments(
+                db_collection_name,
+                sacred_id,
+                list(cancel_states),
+                batch_id,
+                filter_dict,
+                yes=False,
+                confirm_threshold=1,
```

Comment:
Should we not have the default confirm threshold here? Should the deletion terminate (w/o deleting), when the user is not cancelling their experiments?

Reply:
It was meant as a caution since it may be unexpected. Would you trust the user to always know that there are running experiments? If so we can also use the default

Reply:
Yes, good point.